public/app/features/explore/ResponseErrorContainer.test.tsx

```diff
@@ -50,6 +50,14 @@ function setup(error: DataQueryError) {
     series: [],
     state: LoadingState.Error,
     error,
+    graphFrames: [],
+    logsFrames: [],
+    tableFrames: [],
+    traceFrames: [],
+    nodeGraphFrames: [],
+    graphResult: null,
+    logsResult: null,
+    tableResult: null,
```

Comment:
this is needed because of the changes in https://github.com/grafana/grafana/pull/45148/files#diff-70e1cc4ad5a5754fe5e6e51db45d1be0501db91555cbfb6db1fc0f80aa328415R200